compiler/il/OMRNode.cpp

```diff
@@ -2352,6 +2353,47 @@ OMR::Node::isNotCollected()
 
 
 bool
+OMR::Node::canBeInternalPtrAtObject()
```

Comment:
can this please be canBeInternalPtrOfObject for grammatical reasons. Can you please add a document comment describing what the function does as well?